workshop/exercise-5/README.md

```diff
@@ -1,73 +1,68 @@
-
-A Knative BuildTemplate encapsulates a shareable build process with some limited parameterization capabilities.
-
-1. Install the kaniko build template to your cluster.
+### Deploy vnext
+1. Let's deploy vnext, again using a docker image on dockerhub. Maybe we want to slowly roll users over from our old version to the new version, or do some A/B testing of the new version to see what users like better. Let's see what the yaml for this looks like.
 
     ```
-    kubectl apply --filename https://raw.githubusercontent.com/knative/build-templates/master/kaniko/kaniko.yaml
+    cat fib-service2.yaml
     ```
 
-2. Use kubectl to confirm you installed the kaniko build template, as well as to see some more details about it.  You'll see that this build template accepts parameters of `IMAGE` and `DOCKERFILE`.  `IMAGE` is the name of the image you will push to the container registry, and `DOCKERFILE` is the path to the Dockerfile that will be built.
-
-	Command:
-	```
-	kubectl get BuildTemplate kaniko -o yaml
-	```
+    Expected Output:
+    ```
+    apiVersion: serving.knative.dev/v1alpha1
+    kind: Service
+    metadata:
+        name: fib-knative
+        namespace: default
+    spec:
+        release:
+            revisions: ["fib-knative-xxxxx", "@latest"]
+            rolloutPercent: 10
+            configuration:
+                revisionTemplate:
+                    spec:
+                        container:
+                            image: docker.io/ibmcom/fib-knative:vnext
+    ```
 
-	Example Output:
-	```yaml
-      spec:
-        generation: 1
-        parameters:
-        - description: The name of the image to push
-          name: IMAGE
-        - default: /workspace/Dockerfile
-          description: Path to the Dockerfile to build.
-          name: DOCKERFILE
-        steps:
-        - args:
-          - --dockerfile=${DOCKERFILE}
-          - --destination=${IMAGE}
-          image: gcr.io/kaniko-project/executor
-          name: build-and-push
-	```
+	Notice that we've added a rolloutPercent of 10. We've also added our old revision name to the revisions array. Your revision name will be different than `fib-knative-xxxxx`, so you will update this file.
 
+2. Get the name of your first revision.
 
-### Deploy the Fibonacci App Using kubectl and service.yaml
+    ```
+    kubectl get revision
+    ```
 
-1. Edit the `service.yaml` file to point to your own container registry namespace by replacing instances of `<NAMESPACE>` with the container registry namespace you created earlier.
+    Expected Output:
+    ```
+    NAME                SERVICE NAME        GENERATION   READY   REASON
+    fib-knative-rgqjl   fib-knative-rgqjl   1            True    
```

Comment:
weird- I think there's a bug in Kn. Seems like "SERVICE NAME" should be "fib-knative". I'll ask about that

Reply:
Apparently, it's the K8s service name not the KnService name.
https://github.com/knative/serving/pull/4231